hw/bsp/nucleo-f767zi/pkg.yml

```diff
@@ -27,7 +27,8 @@ pkg.keywords:
     - stm32f7
     - nucleo
 
-pkg.cflags: -DSTM32F767xx
+      # pkg.cflags: -DSTM32F767xx
+pkg.cflags: -DSTM32F777xx
```

Comment:
Was this change intentional? A bit unexpected.

Reply:
STM32 is not finished, this is here because F777 supports CRYP and other parts don't. Once this is properly finished I will drop this commit.